src/app/models/propTypes/promo/index.js

```diff
@@ -1,4 +1,5 @@
-import { number, shape, string } from 'prop-types';
+import { number, shape, string, arrayOf } from 'prop-types';
+import { bool } from '@storybook/addon-knobs';
```

Comment:
![](https://media.giphy.com/media/mheYSVOu2jk3K/giphy.gif)

Reply:
![is](https://user-images.githubusercontent.com/8601460/67038253-f431f500-f116-11e9-9eb2-d161a5331d46.png)

Reply:
![](https://media.giphy.com/media/3o7521WVJ7V4IGwIUw/giphy.gif)